RunCat365/Program.cs

```diff
@@ -111,6 +115,83 @@ private static Theme GetSystemTheme()
             return (int)value == 0 ? Theme.Dark : Theme.Light;
         }
 
+        private static async Task<bool> GetStartupAsync() {
+            if (IsRunningAsPackaged()) {
+                if (startupTask is null) startupTask = await StartupTask.GetAsync("RunCatStartup");
+                if (startupTask is null) return false;
+                if (startupTask.State == StartupTaskState.Enabled) return true;
+                return false;
+            } else {
+                var keyName = @"Software\Microsoft\Windows\CurrentVersion\Run";
+                using var rKey = Registry.CurrentUser.OpenSubKey(keyName);
+                if (rKey is null) return false;
+                var value = (rKey.GetValue(Application.ProductName) is not null);
+                rKey.Close();
+                return value;
+            }
+        }
+
+        private static async Task<bool> SetStartupAsync(bool isChecked) {
+            if (IsRunningAsPackaged()) {
```

Comment:
Is it necessary to separate the logic for the Store version and the non-Store version? Will a common logic not work for both? Also, since I have no plans to distribute this outside of the Store, I believe the logic for the non-Store version is unnecessary.